### PR TITLE
Update acta-orthopaedica.csl

### PR DESCRIPTION
**Pull request by gforge**
The Acta Orthopaedica is actually not just Vancouver-author-date but has its own quirks.

### CODE REVIEW

**Comment by POBrien333** [comment]
You'll need to put this file in the normal foulder and not in the dependent folder. You can do this by editing the filename and just hitting backspace and it'll start deleting "dependent" (took me a while to realise that.
Then it'll validate.

File: acta-orthopaedica.csl [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+      <date-part name="day"/>
+    </date>
+    <terms>
+      <term name="retrieved">available</term>
+      <term name="section" form="short">sect.</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name delimiter-precedes-last="always" et-al-min="19" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
```
Comment (adam3smith) [inline acta-orthopaedica.csl]: delete the et-al-min here. If the setting in the bibliography (70/60) isn't right, move it to the bibliography.

File: acta-orthopaedica.csl [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+          <text variable="publisher-place" prefix="[" suffix="]"/>
+        </if>
+        <else>
+          <text variable="publisher-place"/>
+        </else>
+      </choose>
+      <text variable="publisher"/>
+    </group>
+  </macro>
+  <macro name="access">
```
Comment (adam3smith) [inline acta-orthopaedica.csl]: you're not using this macro -- delete it.

**Comment by rmzelle** [comment]
Thanks!

**Comment by gforge** [comment]
Thanks for the help fixing the remaining issues! You should by the way highlight more clearly the CSL language primer http://docs.citationstyles.org/en/stable/primer.html in the editing styles section. It is currently not directly linked and the CSL docs are hidden at the end of the paragraph. I'm a sloppy reader and despite reading the section several times I've failed to navigate to this section.

**Comment by rmzelle** [comment]
> in the editing styles section.

Where is this section? http://citationstyles.org/ or here on the GitHub wiki?

**Pull request merged**